packages/veritone-react-common/src/components/Breadcrumbs/index.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { shape, string, number, arrayOf, oneOfType, node, func } from 'prop-types';
+import { MenuList, Popover } from '@material-ui/core';
+import { ChevronRight, MoreVert } from '@material-ui/icons';
+import classNames from 'classnames';
+
+import BreadcrumbItem from './BreadcrumbItem';
+
+import styles from './Breadcrumbs.scss';
+
+export default class Breadcrumbs extends React.Component {
+  static propTypes = {
+    pathList: arrayOf(shape({
+      id: string.isRequired,
+    })),
+    maxItems: number,
+    seperator: oneOfType([string, node]),
+    onCrumbClick: func.isRequired
+  }
+
+  static defaultProps = {
+    maxItems: 5,
+    seperator: <ChevronRight className={styles['icon-color']} />,
+  }
+
+  state = {
+    openSpreadPath: false
+  }
+
+  onSpreadClick = (event) => {
+    this.anchorEl = event.target;
+    this.setState(({ openSpreadPath }) => ({
+      openSpreadPath: !openSpreadPath
+    }));
+  }
+
+  clickAway = () => {
+    this.anchorEl = null;
+    this.setState({
+      openSpreadPath: false
+    })
+  }
+
+  onCrumbClick = (event) => {
+    let elementData = Object.assign({}, event.target.dataset);
+    if (Object.keys(elementData).length === 0) {
+      elementData = Object.assign({}, event.target.parentNode.dataset);
+    }
+    this.props.onCrumbClick(elementData);
+  }
+
+  handleClose = (event) => {
+    event.stopPropagation();
+  }
+
+  render() {
+    const {
+      pathList,
+      maxItems,
+      seperator,
+    } = this.props;
+
+    const {
+      0: rootCrumb,
+      1: secondCrumb,
+      [pathList.length - 1]: lastCrumb
+    } = pathList;
+    const hiddenCrumbs = pathList.slice(2, pathList.length - 1)
+
+    return (
+      <div className={styles['breadcrumb-container']}>
+        {
+          pathList.length < maxItems ? (
+            pathList.map((crumb, index) => (
+              <React.Fragment key={crumb.id}>
+                {index > 0 && seperator}
+                <BreadcrumbItem
```

Comment:
Missing the briefcase icon for the root crumb

Reply:
@atb91590 For now, I woud like to parsing icon via pathList props. eg: `pathList=[{id: 'root', icon: <Work />}]`. I will update the story file